build/builder.go

```diff
@@ -334,25 +333,14 @@ func (o *Options) SetDefaults() {
 	}
 }
 
-func hashString(s string) string {
-	h := sha1.New()
-	_, _ = io.WriteString(h, s)
-	return fmt.Sprintf("%x", h.Sum(nil))
-}
-
 // ShardName returns the name the given index shard.
 func (o *Options) shardName(n int) string {
 	return o.shardNameVersion(zoekt.IndexFormatVersion, n)
 }
 
 func (o *Options) shardNameVersion(version, n int) string {
 	prefix := url.QueryEscape(cmp.Or(o.ShardPrefix, o.RepositoryDescription.Name))
```

Comment:
I think we're now calling `url.QueryEscape` twice, once here and once in `zoekt.ShardName`.

Reply:
🤦 thanks!